src/draw/lv_draw_vector.c

```diff
@@ -790,6 +790,8 @@ void lv_vector_dsc_skew(lv_vector_dsc_t * dsc, float skew_x, float skew_y)
 
 void lv_vector_for_each_destroy_tasks(lv_ll_t * task_list, vector_draw_task_cb cb, void * data)
 {
+    if(task_list == NULL) return;
```

Comment:
task_list being NULL is something okay to happen, or this is just a protection against a catastrophic failure? In later case, I suggest to always put a LOG_WRN for better debugging experience

Reply:
It's a workaround for a not perfectly clear life cycle management. Various parts of the vector descriptors are freed at various stages, so it can heppen that the things here are already freed. 

We should have a larger refactor here in the future.

Reply:
I see it, even though it still something to handle, it is part of larger problem not being handled here, I'm okay to resolve it for now.